solutions/ai-chatgpt/README.md

```diff
@@ -26,7 +26,11 @@ npx create-next-app --example https://github.com/vercel/examples/tree/main/solut
 yarn create next-app --example https://github.com/vercel/examples/tree/main/solutions/ai-chatgpt
 ```
 
-Next, run Next.js in development mode:
+## Running Locally
+
+After cloning the repo, go to [OpenAI](https://beta.openai.com/account/api-keys) to make an account and put your API key in a variable `OPENAI_API_KEY` in a file called `.env.local`.
+
+Then, run the application in the command line and it will be available at `http://localhost:3000`.
```

Comment:
Do this like other examples that use `.env.example` do it, they provide instructions on how to clone the file and the dev server running is mentioned after the command.

Reply:
done